### PR TITLE
Update/ Insert Modal

### PR DESCRIPTION
**Pull request by ebadowski**

### CODE REVIEW

File: models/ratings.js [hunk 1]
```diff
@@ -24,23 +24,35 @@ var ratings = {
...
     );
   },
-  rateRecipe: function(req, res) {
+  rateRecipe: function (req, res) {
     orm.create(
       {
         table: "ratings",
         values: {
-          "rating_id": req.params.id,
+          "recipe_id": req.params.id,
```
Comment (clarkalms) [inline models/ratings.js:37]: this should stay as "rating_id" because we're adding it from the recipe id in the api route params.

Comment (ebadowski) [reply models/ratings.js:37]: It's editing the recipe_id column in the ratings table, rating_id is auto incremented

File: public/js/recipeModal.js [hunk 2]
```diff
@@ -81,159 +81,92 @@ var testRatingArr = [
 /////      UPPER CASE ARE TO DO ITEMS     /////
 ///////////////////////////////////////////////
 
-
```
Comment (clarkalms) [inline public/js/recipeModal.js:84]: @ljmccode  should review these changes since she was the one who wrote this code.

**Review by clarkalms — COMMENTED** [review]
Please see comments

File: models/ratings.js (hunk 1 above)
Comment (clarkalms) [reply models/ratings.js:37]: There isn't a "recipe_id" column in ratings table

Comment (ebadowski) [reply models/ratings.js:37]: we added it in after class yesterday, was part of the foreign key fix

Comment (clarkalms) [reply models/ratings.js:37]: copy.

**Review by clarkalms — COMMENTED** [review]
I approved the changes to the orm/api files. Slack @ljmccode to get her to approve other changes to public files

File: public/js/recipeModal.js (hunk 2 above)
Comment (ebadowski) [reply public/js/recipeModal.js:84]: This was all me too, just organized the code, and updated references to make the values for the AJAX calls to work

**Review by clarkalms — APPROVED** [review]

**Comment by clarkalms** [comment]
Changes are all good.

**Pull request merged**